README.md

```diff
@@ -40,32 +40,21 @@ The [Authentication template](https://www.npmjs.com/package/@brightlayer-ui/cra-
 
 ## Usage
 
-You can use these templates with the Brightlayer UI CLI (recommended):
+You can use these templates with the Brightlayer-UI NPM starter package:
```

Comment:
I don't know what 'NPM starter package' means. Maybe something like:
```suggestion
You can use these templates with the `@brightlayer-ui/create-blui-react-app` package:
```

You may also want to show how to do it with yarn since that is still our preferred package manager.